app/src/main/java/com/mapbox/vision/examples/activity/main/MainActivity.kt

```diff
@@ -346,9 +350,31 @@ class MainActivity : AppCompatActivity() {
         }
     }
 
+    private val REPLAY_SESSION_PATH by lazy {
+        "${application.getExternalFilesDir(null)!!.absolutePath}/Replays/"
+    }
+    private val RECORD_SESSION_PATH by lazy {
+        "${application.getExternalFilesDir(null)!!.absolutePath}/Records/"
+    }
+
     private fun tryToInitVisionManager() {
         if (isPermissionsGranted && !visionManagerWasInit) {
-            VisionManager.create()
+
+            val replay = VisionManager.VisionMode.SessionReplay(
+                path = REPLAY_SESSION_PATH
+            )
+            replay.setSessionProgress(60000)
```

Comment:
why 60k?) 60k mils -> 1 min?

Reply:
I'll remove this code before merge - it's only for demonstration & discussions.
PR is DO NOT MERGE now.